dp/core/dpdk.c

```diff
@@ -59,7 +59,6 @@ int dpdk_init(void)
 	const int pool_size = 32768;
 
 	optind = 0;
-	internal_config.no_hugetlbfs = 1;
```

Comment:
Apparently the initialization of the i40e in DPDK requires huge pages, so I understand the reason for this change. However, I've noticed that the boot time of IX has increased after removing this line. Can you investigate if we can initialize the i40e without removing this line?